docs/source/contributors/index.md

```diff
@@ -6,14 +6,7 @@ This page is intended for people interested in building new or modified function
 
 You can develop Jupyter AI on any system that can run a supported Python version up to and including 3.10, including recent Windows, macOS, and Linux versions. Python 3.11 is **not supported** due
 to incompatibility with the [ray](https://pypi.org/project/ray/) library that we use.
-If you are using an Apple Silicon-based Mac (M1, M1 Pro, M2, etc.), and you see an error when using
-Python 3.10, try installing version 3.9 instead.
-
-If you use `conda`, you can install Python 3.10 in your environment by running:
-
-```
-conda install python=3.10
```

Comment:
Please reinstate instructions about how to use 3.10, because 3.11 is not fully supported by Ray on any OS.